examples/webgl_loader_sea3d_hierarchy.html

```diff
@@ -120,6 +116,9 @@
 				renderer.setPixelRatio( window.devicePixelRatio );
 				renderer.setSize( window.innerWidth, window.innerHeight );
 				renderer.setClearColor( 0x333333, 1 );
+				renderer.shadowMap.enabled = true;
+				renderer.shadowMap.type = THREE.PCFSoftShadowMap;
+				renderer.shadowMap.cullFace = THREE.CullFaceBack;
```

Comment:
@tschw In unification of `mixers` I had a problem. 
`AnimationMixer` works with unique names of `AnimationClip` but the variable `name` not represents an `uuid` semantically. If rename `name` to `uuid` or `ns` open space to declare the real animation name in `name` slot like `idle`, `walk`, `run`, etc.

https://github.com/mrdoob/three.js/blob/dev/src/animation/AnimationClip.js#L11

For example in skinning example I have:

https://github.com/mrdoob/three.js/blob/dev/examples/webgl_loader_sea3d_skinning.html#L171

```javascript
if (player.currentAnimation.name == "idle")
```

`currentAnimation` is an `AnimationClip`. I could assign `currentAnimation.ns` or `currentAnimationName` but I think it would be clearer this way `currentAnimation.name` and `currentAnimation.uuid` for clip name currently.

In this example I had the same problem by the names conflict.

https://github.com/mrdoob/three.js/blob/dev/examples/webgl_loader_sea3d_keyframe.html